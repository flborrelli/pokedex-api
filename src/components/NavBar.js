import React from "react";
import { Navbar } from "react-bootstrap";

function NavBar() {
  return (
    <Navbar className="mynavbar d-flex justify-content-center">
      <Navbar.Brand href="#" className='logo mr-4'>
        <img
          src="/images/pokeapi.png"
          className="nav-img d-inline-block align-top"
          alt=""
        />
      </Navbar.Brand>
      <h2 className="navbar-title ml-3">Find your favourites Pokemons</h2>
    </Navbar>
  );
}

export default NavBar;
