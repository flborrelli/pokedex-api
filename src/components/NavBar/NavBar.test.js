import React from 'react';
import NavBar from './NavBar';
import { create } from 'react-test-renderer';

  test('should match with navbar snapshot', () => {
    const navComponent = create(<NavBar />);
    expect(navComponent.toJSON()).toMatchSnapshot();
  });