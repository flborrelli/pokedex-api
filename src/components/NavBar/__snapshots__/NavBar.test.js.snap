// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavBar /> should match with navbar snapshot 1`] = `
<nav
  className="mynavbar d-flex justify-content-center navbar navbar-expand navbar-light"
  data-testid="navbar"
>
  <a
    className="logo mr-4 navbar-brand"
    href="#"
  >
    <img
      alt=""
      className="nav-img d-inline-block align-top"
      src="/images/pokeapi.png"
    />
  </a>
  <h2
    className="navbar-title ml-3"
  >
    Find your favourites Pokemons
  </h2>
</nav>
`;

exports[`<Navbar /> should match with snapshot 1`] = `
<nav
  className="mynavbar d-flex justify-content-center navbar navbar-expand navbar-light"
  data-testid="navbar"
>
  <a
    className="logo mr-4 navbar-brand"
    href="#"
  >
    <img
      alt=""
      className="nav-img d-inline-block align-top"
      src="/images/pokeapi.png"
    />
  </a>
  <h2
    className="navbar-title ml-3"
  >
    Find your favourites Pokemons
  </h2>
</nav>
`;

exports[`should match with navbar snapshot 1`] = `
<nav
  className="mynavbar d-flex justify-content-center navbar navbar-expand navbar-light"
  data-testid="navbar"
>
  <a
    className="logo mr-4 navbar-brand"
    href="#"
  >
    <img
      alt=""
      className="nav-img d-inline-block align-top"
      src="/images/pokeapi.png"
    />
  </a>
  <h2
    className="navbar-title ml-3"
  >
    Find your favourites Pokemons
  </h2>
</nav>
`;
